android/src/test/java/com/amplitude/android/AmplitudeTest.kt

```diff
@@ -70,13 +74,14 @@ class AmplitudeTest {
         amplitudeDispatcherField.set(amplitude, dispatcher)
     }
 
-    private fun createConfiguration(minTimeBetweenSessionsMillis: Long? = null): Configuration {
+    private fun createConfiguration(minTimeBetweenSessionsMillis: Long? = null, storageProvider: StorageProvider = InMemoryStorageProvider()): Configuration {
```

Comment:
If we keep the test as file storage, anything is failed?

Reply:
I see. I wonder if we clear the storage file then file storage could work too.

Reply:
Yes, it works - I've checked.

Reply:
Cool, I guess it is fine now and we could optimize the test case in later changes